kuma/wiki/constants.py

```diff
@@ -201,6 +201,7 @@
 
 DIFF_WRAP_COLUMN = 65
 EXPERIMENT_TITLE_PREFIX = 'Experiment:'
+DEPRECATED_PAGE_NAMESPACES = ('User:', 'Talk:', 'User_talk:', 'Template_talk:')
```

Comment:
Nit: It would be nice if there was one source for all of these namespaces. Now we have a list is kuma/wiki/views/legacy.py:

https://github.com/mozilla/kuma/blob/2ee1e66dec001d4ddf1c9fa0a32929d33faca39c/kuma/wiki/views/legacy.py#L13-L23

and a slightly different list in managers.py:

https://github.com/mozilla/kuma/blob/2ee1e66dec001d4ddf1c9fa0a32929d33faca39c/kuma/wiki/managers.py#L26-L33

and in search.py:

https://github.com/mozilla/kuma/blob/2ee1e66dec001d4ddf1c9fa0a32929d33faca39c/kuma/wiki/search.py#L30-L31